deploy/cloud-info/backend.tf

```diff
@@ -1,11 +1,3 @@
-# This is where the info about the deployment is to be stored
```

Comment:
Please bring this back

Reply:
Unintentional change added with `git add -u`, sorry.
Thanks for spotting, I am reverting.